md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/UserRolesAPI.java

```diff
@@ -55,7 +58,14 @@ public RestWrapper list(
             @PathVariable("uname") String username, Principal principal) {
         RestWrapper restWrapper = null;
         try {
-            List<com.wipro.ats.bdre.md.dao.jpa.UserRoles> jpaUserRolesList = userRolesDAO.listByName(username);
+
+            List<com.wipro.ats.bdre.md.dao.jpa.UserRoles> jpaUserRolesList = new ArrayList<>();
+            LOGGER.info("username is "+username+"principal.getname "+principal.getName());
+            if (!username.equals("cur_user"))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "cur_user" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1132)